module/platform/Webots/src/Webots.hpp

```diff
@@ -77,9 +77,11 @@ namespace module::platform {
         uint32_t current_sim_time = 0;
         /// @brief The current real time in milliseconds (unix time)
         uint64_t current_real_time = 0;
-        /// @brief Interpolation factor to smooth clock. 0.0 is no smoothing (raw updates from webots), 1.0 takes no
-        /// updates from webots
+        /// @brief Interpolation factor to smooth clock. 0.0 is no smoothing (raw updates from Webots), 1.0 takes no
+        /// updates from Webots
         double clock_smoothing = 0.0;
```

Comment:
This however is a config value (if there is a config struct in this reactor otherwise it's fine)